examples/create_rosa_cluster/create_rosa_sts_cluster/main.tf

```diff
@@ -37,42 +37,11 @@ locals {
   sts_roles = {
       role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Installer-Role",
       support_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Support-Role",
-      operator_iam_roles = [
-        {
-          name =  "cloud-credential-operator-iam-ro-creds",
-          namespace = "openshift-cloud-credential-operator",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cloud-credential-operator-cloud-c",
-        },
-        {
-          name =  "installer-cloud-credentials",
-          namespace = "openshift-image-registry",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-image-registry-installer-cloud-cr",
-        },
-        {
-          name =  "cloud-credentials",
-          namespace = "openshift-ingress-operator",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-ingress-operator-cloud-credential",
-        },
-        {
-          name =  "ebs-cloud-credentials",
-          namespace = "openshift-cluster-csi-drivers",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cluster-csi-drivers-ebs-cloud-cre",
-        },
-        {
-          name =  "cloud-credentials",
-          namespace = "openshift-cloud-network-config-controller",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-cloud-network-config-controller-c",
-        },
-        {
-          name =  "aws-cloud-credentials",
-          namespace = "openshift-machine-api",
-          role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.operator_role_prefix}-openshift-machine-api-aws-cloud-credentials",
-        },
-      ]
       instance_iam_roles = {
         master_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-ControlPlane-Role",
         worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/ManagedOpenShift-Worker-Role"
-      },    
+      },
+      operator_role_prefix = var.operator_role_prefix,
```

Comment:
Why do you have to set the local parameter, as you already have the variable?

Reply:
I need to inject it in somehow  to the cluster rosa resource 
I keep it as the other attribute - to be in STS